app/views/users/index.html.erb

```diff
@@ -1 +1,18 @@
-<h1>List of all users</h1>
\ No newline at end of file
+<h1 class ="usersTitle">List of all users</h1>
+<div class= "allUsers">
+  <% @users.each do |user| %>
+  <div class= "container">
+  <div class= "photo">
+    <%= image_tag user.Photo , class: "img-profile" %>
+  </div>
+  <div class="nameAndPosts">
+    <p class = "name">
+      <%= link_to user.Name, user_path(user) %>
+    </p>
+    <p class = "numberOfPosts" >
+      Number of posts: <%= user.PostCounter %>
+    </p>
+  </div>
+  </div>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you use `partials` for rails best best practices. As, with a `partial`, you can move the code for rendering a particular piece of a response to its own file instead of repeating the same code every time. I suggest you **[refer this article for more details about partials](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials)**.
I recommend you to also apply this approach to your other views pages such as `Post`, `Comment`,...

Reply:
Hi, noted and applied, thank you for the recommendation, now my code is shorter.